File: java/src/phase 1-conditional_thinking_java/Q18AlphabetRange.java
import java.util.Scanner;

/**
 * Phase 1: Conditional Thinking (If-Else, Boolean Logic)
 * Level 2: Nested If & Multiple Conditions
 * Question 18: Take an alphabet character and check if it lies between 'a' and 'm' or 'n' and 'z'.
 * 
 * Mentor Tips (Java 26):
 * - You can compare characters directly using '<=' and '>=' (comparison is lexicographical).
 */
public class Q18AlphabetRange {

    public static String solve(char charVal) {
        // TODO: Implement this method to solve the question
        return "";
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Char: "); char c = scanner.next().charAt(0);
                System.out.println("Range: " + solve(c));
        scanner.close();
    }
}

File: python/phase 1-condtional_thinking/q18_alphabet_range.py
"""
Phase 1: Conditional Thinking (If-Else, Boolean Logic)
Level 2: Nested If & Multiple Conditions
Question 18: Take an alphabet character and check if it lies between 'a' and 'm' or 'n' and 'z'.

Mentor Tips (Python 3.14+):
- You can compare characters directly using '<=' and '>=' (comparison is lexicographical).
"""

def solve(char: str) -> str:
    # TODO: Implement this method to solve the question
    return "a-m" # or "n-z" or "outside"

def main():
    char = input("Char: ")\n    print(f"Range: {solve(char)}")

if __name__ == "__main__":
    main()
